docs/pages/api-reference/liveblocks-emails.mdx

```diff
@@ -6,111 +6,840 @@ meta:
+
+        // The formatted comment, pass it to React `children`
+        reactBody: { /* ... */ },
+
+        author: {
+          id: "aurelien@example.com",
+          info: { /* Custom user info you have resolved */ },
+        },
+      },
+    }
+    ```
+
+    <div className="-mt-2">
+
+    ```js title="Unread replies" isCollapsable isCollapsed
+    {
+      type: "unreadReplies",
+      roomId: "my-room-id",
+
+      // Unread replies means multiple comments
+      comments: [
+        {
+          id: "cm_asfs8f...",
+          threadId: "th_sj30as..."
+          createdAt: Date <Fri Dec 15 2023 14:15:22 GMT+0000 (Greenwich Mean Time)>,
+
+          // The formatted comment, pass it to React `children`
+          reactBody: { /* ... */ },
+
+          author: {
+            id: "aurelien@example.com",
+            info: { /* Custom user info you have resolved */ },
+          },
+        },
+
+        // More comments
+        //...
+      ],
+    }
+    ```
+
+    </div>
 
-Prepares data from a
-[`ThreadNotificationEvent`](/docs/platform/webhooks#Thread-notification) and
-convert comment bodies as an html safe string.
+  </PropertiesListItem>
+</PropertiesList>
 
-It returns a `ThreadNotificationEmailDataAsHtml` or `null` if there are no
-unread comments.
+<PropertiesList title="Arguments">
+  <PropertiesListItem name="client" type="Liveblocks" required>
+    A [`Liveblocks`](/docs/api-reference/liveblocks-node#Liveblocks-client)
+    Node.js client.
+  </PropertiesListItem>
+  <PropertiesListItem name="event" type="ThreadNotificationEvent" required>
+    An event from a
+    [`ThreadNotificationWebhook`](/docs/platform/webhooks#Thread-notification).
+  </PropertiesListItem>
+  <PropertiesListItem name="options" type="object">
+    A number of options to customize the format of the comments, adding user
+    info, room info, and styles.
+  </PropertiesListItem>
+  <PropertiesListItem
+    name="options.resolveUsers"
+    detailedType='async? (args: ResolveUsersArgs) => (UserMeta["info"] | undefined)[] | undefined'
+  >
+    A function that resolves user information in
+    [Comments](/docs/products/comments). Return an array of `UserMeta["info"]`
+    objects in the same order they arrived. Works similarly to the [resolver on
+    the
+    client](/docs/api-reference/liveblocks-react#LiveblocksProviderResolveUsers).
+    [Learn more](#prepare-thread-notification-email-as-react-resolving-data).
+  </PropertiesListItem>
+  <PropertiesListItem
+    name="options.resolveRoomsInfo"
+    detailedType="async? (args: ResolveRoomsInfoArgs) => (RoomInfo | undefined)[] | undefined"
+  >
+    A function that resolves room information. Return an array of `RoomInfo`
+    objects in the same order they arrived. Works similarly to the [resolver on
+    the
+    client](/docs/api-reference/liveblocks-react#LiveblocksProviderResolveRoomsInfo).
+    [Learn more](#prepare-thread-notification-email-as-react-resolving-data).
+  </PropertiesListItem>
```

Comment:
Here it's not the same. It's `resolveRoomInfo` only and it returns an optional promise of `DRI` or `undefined`, not an array.

It's made this way because a notification event (thread or text mention) is only for one room.

Reply:
My mistake! Fixed.